### PR TITLE
Enables implicit conversion for python algorithm

### PR DESCRIPTION
**Pull request by AlexCatarino**
- Updates pythonnet package
This update implemented implicit conversion.
From now on, python algorithms will be able to use string object instead of Symbol object when a method overload requires Symbol.

- Modifies python example algorithms to show implicit conversion benefits

### CODE REVIEW

File: Algorithm.Python/CustomChartingAlgorithm.py [hunk 1]
```diff
@@ -56,7 +56,11 @@ def Initialize(self):
         self.resamplePeriod = (self.EndDate - self.StartDate) / 2000
         
     def OnData(self, slice):
-        self.lastPrice = slice[self.spy].Close
+        if slice["SPY"] is None:
```
Comment (jaredbroad) [inline Algorithm.Python/CustomChartingAlgorithm.py]: 👍

File: Algorithm.Python/CustomDataBitcoinAlgorithm.py [hunk 2]
```diff
@@ -41,18 +41,17 @@ def Initialize(self):
 
         # Define the symbol and "type" of our generic data:
         self.AddData(Bitcoin, "BTC")
-        self.btc = self.Securities["BTC"].Symbol
-
+        
 
     def OnData(self, data):
-        if self.btc not in data: return
+        if "BTC" not in data: return
```
Comment (jaredbroad) [inline Algorithm.Python/CustomDataBitcoinAlgorithm.py]: Does the symbol method still work as well?

File: Algorithm.Python/DailyAlgorithm.py [hunk 3]
```diff
@@ -11,11 +11,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import clr
-clr.AddReference("System")
-clr.AddReference("QuantConnect.Algorithm")
-clr.AddReference("QuantConnect.Indicators")
-clr.AddReference("QuantConnect.Common")
+from clr import AddReference
```
Comment (jaredbroad) [inline Algorithm.Python/DailyAlgorithm.py]: What about the rest of the CLR, you don't need that?

Comment (AlexCatarino) [inline Algorithm.Python/DailyAlgorithm.py]: No. In fact, so far we haven't used any methods other than AddReference.

File: Algorithm.Python/CustomDataBitcoinAlgorithm.py (hunk 2 above)
Comment (AlexCatarino) [inline Algorithm.Python/CustomDataBitcoinAlgorithm.py]: Yes. I made it work for both cases.

**Pull request merged**